config/crds/apis.kcp.dev_apiexports.yaml-patch

```diff
@@ -8,7 +8,5 @@
 - op: add
   path: /spec/versions/name=v1alpha1/schema/openAPIV3Schema/properties/spec/properties/permissionClaims/items/x-kubernetes-validations
   value:
-    - rule: "(has(self.group) && self.group != '') || self.resource in ['configmaps', 'namespaces', 'secrets', 'serviceaccounts']"
-      message: .resource must be one of configmaps, namespaces, secrets, serviceaccounts if .group is empty.
-    - rule: "!(has(self.group) && self.group != '') || (has(self.identityHash) && self.identityHash != '')"
-      message: .identityHash must be set if .group is not empty.
\ No newline at end of file
+    - rule: "!(has(self.group) && self.group != '' && !self.group.endsWith('.k8s.io')) || (has(self.identityHash) && self.identityHash != '')"
+      message: .identityHash must be set if .group is not empty or *.k8s.io.
```

Comment:
we cannot have `*.k8s.io` here. Why not list all resources if care about these validations?

Reply:
Was chatting with @sttts - we should probably just remove this entire validation for the time being, unless we can easily list out every possible group-resource-identityHash.

Reply:
discussed briefly with Andy. I agree, we should really remove these validations because we have a hard time to make them complete and correct.

Also notice that every change in resources we allow for claim is technically a breaking change, either in upgrade or in downgrade. Hence, technically we would have to use ratcheting validation here which makes it even more complicated and hard to reason about.

Reply:
I can remove these entirely.